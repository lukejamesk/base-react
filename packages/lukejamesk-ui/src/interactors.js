export { default as ButtonBaseInteractor } from './ButtonBase/ButtonBase.interactor'
export { default as ExpanderInteractor } from './Expander/Expander.interactor'
export { default as ExpanderHeaderInteractor } from './Expander/ExpanderHeader.interactor'
export { default as ExpanderBodyInteractor } from './Expander/ExpanderBody.interactor'
