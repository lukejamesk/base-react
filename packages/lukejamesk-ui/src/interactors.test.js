import { testPackageExports } from '@lukejamesk/test-utils'
import * as lukejameskInteractors from './interactors'

testPackageExports('@lukejamesk/core/interactors', lukejameskInteractors)
