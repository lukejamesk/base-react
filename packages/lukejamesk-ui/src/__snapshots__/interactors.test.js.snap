// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@lukejamesk/core/interactors should match snapshot 1`] = `
Array [
  "ButtonBaseInteractor",
  "ExpanderBodyInteractor",
  "ExpanderHeaderInteractor",
  "ExpanderInteractor",
]
`;
